tools/cmock/create_test.cmake

```diff
@@ -109,10 +122,42 @@ function(create_mock_list mock_name mock_list cmock_config)
     endforeach()
     target_include_directories(${mock_name} PUBLIC
                                ${mocks_dir}
+                               ${mock_include_list}
            )
     set_target_properties(${mock_name} PROPERTIES
                         LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lib
                         POSITION_INDEPENDENT_CODE ON
             )
+    target_compile_definitions(${mock_name} PUBLIC
+            ${mock_define_list}
+        )
     target_link_libraries(${mock_name} libcmock.a libunity.a)
 endfunction()
+
+
+function(create_real_library target
+                             src_file
+                             real_include_list
+                             mock_name)
+    add_library(${target} STATIC
+            ${src_file}
+        )
+    target_include_directories(${target} PUBLIC
+            ${real_include_list}
+            )
+    set_target_properties(${target} PROPERTIES
+                COMPILE_FLAGS " \
```

Comment:
It might be desirable to also add -Wpedantic and -Wextra for the real code. What are your thoughts on this?

Reply:
sounds good, i will add them, there is a failing check I will fix both